llvm/lib/Analysis/InlineAdvisor.cpp

```diff
@@ -63,6 +65,65 @@ static cl::opt<bool>
 
 extern cl::opt<InlinerFunctionImportStatsOpts> InlinerFunctionImportStats;
 
+/// Flag to path of dynamic libarary
+class DLInlineAdvisorInfo {
+  typedef std::unique_ptr<InlineAdvisor> (*DLInlineAdivsorFactory_t)(
+      Module &M, FunctionAnalysisManager &FAM, LLVMContext &Context,
+      std::unique_ptr<InlineAdvisor> OriginalAdvisor, InlineContext IC,
+      std::string &DLCTX);
+
+  cl::opt<std::string> *Path;
```

Comment:
Is `Path` ever supposed to be null? If not maybe `&Path`?

Reply:
No it shouldn't be, I'll do that